pallets/dapps-staking/src/benchmarking.rs

```diff
@@ -158,7 +158,7 @@ benchmarks! {
         let (developer, contract_id) = register_contract::<T>(1)?;
         let stakers = prepare_bond_and_stake::<T>(1, &contract_id, SEED)?;
         let staker = stakers[0].clone();
-        let stake_amount = BalanceOf::<T>::max_value() / 2u32.into();
+        let stake_amount = Balance::max_value() / u128::from(2u32);
```

Comment:
```suggestion
        let stake_amount = Balance::max_value() / 2u128;
```

Same to following usages.